app.conf.ctmpl

```diff
@@ -0,0 +1,21 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $dataSource := vault (printf "secret/dsde/avram/%s/dbcDataSource" $environment)}}
```

Comment:
You missed the "p" if you were going for "dbcpDataSource" in the vault key. However, there isn't anything DBCP-specific that we're putting in vault so I think this should just be "dataSource" or "db" or something like that.